py-polars/polars/internals/series/datetime.py

```diff
@@ -189,25 +189,57 @@ def minute(self) -> pli.Series:
 
         """
 
-    def second(self) -> pli.Series:
+    def second(self, fractional: bool = False) -> pli.Series:
         """
-        Extract the seconds the from underlying DateTime representation.
+        Extract seconds from underlying DateTime representation.
 
-        Can be performed on Datetime.
+        Can be performed on Datetime columns.
 
-        Returns the second number from 0 to 59.
+        Returns the integer second number from 0 to 59, or a floating
+        point number from 0 < 60 if `fractional=True` that includes
```

Comment:
nit: double ticks would render as code in the HTML documentation

Reply:
Good catch - [fixed](https://github.com/pola-rs/polars/pull/4829/commits/256944e35ab21a194e4b632b7310609b73e9e19f).